engine/wait_for_test_mysql_start.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-until nc -z -v -w30 mysql_test 3306
+until nc -z -v -w30 "${DATABASE_HOST:mysql_test}" 3306
```

Comment:
this won't work on bash, see [stackoverflow](https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash) or experiment:
```
echo ${NON_EXISTANT_VAR:default}
zsh: unrecognized modifier `d'
echo ${NON_EXISTANT_VAR:-default}
default
echo ${NON_EXISTANT_VAR:=default}
default
```